src/network/networkd-address.h

```diff
@@ -61,6 +61,9 @@ struct Address {
 
         /* Called when address become ready */
         address_ready_callback_t callback;
+
+        /* NetLabel */
```

Comment:
that is one useless comment. If the comment consists of just the name of the variable with different capitalization, maybe drop the comment.

Reply:
Dropped the comment and merged the variable definition with another.